src/main/java/com/epam/spring/homework3/repository/ClientRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.homework3.repository;
+
+import com.epam.spring.homework3.model.Client;
+
+import java.util.List;
+
+public interface ClientRepository {
+    Client getClient(int id);
+
+    List<Client> clientsList();
```

Comment:
nit: if you have a getter for a single client called like 'get*', then getter for clients list should also follow the same pattern for readability